src/warc2zim/content_rewriting/css.py

```diff
@@ -23,14 +23,30 @@ def rewrite(self, content: str | bytes) -> str:
             rules = parse_stylesheet(content)
         self.process_list(rules)
 
-        output = serialize(rules)
+        try:
+            output = serialize(rules)
+        except Exception:
+            import sys  # Import sys only in case of error
```

Comment:
I'm a real beginner at this, but does it really make much of a difference (compared to importing once at the module level)? I find it pretty weird (especially since you have to add a comment)